deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -84,7 +84,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v2.0.1
+          image: quay.io/k8scsi/csi-snapshotter:v2.1.0
```

Comment:
Can you update these to the latest patch versions? v2.1.1 for both

Reply:
@ggriffiths I have addressed the changes Please take a look